docs/debug.md

```diff
@@ -15,3 +15,25 @@ aea -s run
 ```
 
 For more guidance on how to use `pdb` check out <a href="https://docs.python.org/3/library/pdb.html" target="_blank">the documentation</a>.
+
+## Using an IDE:
+
+- For VSCode modify the `launch.json` to include the following information:
+
+``` json
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "aea run",
+            "type": "python",
+            "request": "launch",
+            "program": "PATH_TO_VIRTUAL_ENV_PACKAGES/aea/cli/__main__.py",
+            "args": ["-v","DEBUG","--skip-consistency-check","run"],
+            "cwd": "CWD",
+            "console": "integratedTerminal"
+        }
+    ]
+}
+```
+where `PATH_TO_VIRTUAL_ENV_PACKAGES` and `CWD` need to be replaced with the path to the virtual environment's packages and current working directory respectively.
```

Comment:
Maybe it's helpful to be more specific about the CWD?

"where `PATH_TO_VIRTUAL_ENV_PACKAGES` should be replaced with the path to the virtual environment's packages and `CWD` with the working directory for the agent to debug (path to the `aea-config.yaml` file)."